src/data/graphql/typeDefs.ts

```diff
@@ -124,7 +124,7 @@ export default gql`
 
   extend type User {
     reputation(colonyAddress: String!, domainId: Int): String!
-    tokens: [Token!]!
+    tokens(walletAddress: String): [Token!]!
```

Comment:
Can we query tokens without walletAddress?

Reply:
for the user no, you are right - good catch.